packages/http-client-java/emitter/src/common/formats/xml.ts

```diff
@@ -0,0 +1,10 @@
+import { SerializationFormat } from "@autorest/codemodel";
+
+export interface XmlSerializationFormat extends SerializationFormat {
+  name?: string;
+  namespace?: string;
+  prefix?: string;
+  attribute: boolean;
+  wrapped: boolean;
+  text: boolean;
```

Comment:
could we make the name `wrapped` and `text` more clear, or at least add some comment/description to them? Currently from the name, I cannot understand the meaning easily.

Reply:
This is a copy from m4

Detailed comments in `getXmlSerializationFormat` func